src/ir/proto/sql/decode.cc

```diff
@@ -35,4 +36,12 @@ Value DecodeSourceTableColumn(
       decoder_context.GetOrCreateStorage(column_path)));
 }
 
+Value DecodeLiteral(
+    const Literal &literal, DecoderContext &decoder_context) {
+  const absl::string_view literal_str = literal.literal_str();
+  CHECK(!literal_str.empty()) << "required field literal_str was empty.";
+  return Value(value::OperationResult(
+      decoder_context.MakeLiteralOperation(literal_str), "out"));
```

Comment:
Perhaps in a separate PR, I think it will be useful to have some default names for singleton results like `"out"`

Reply:
I've moved all of the names that I'm using for SQL IR construction into `constexpr` globals in the `DecoderContext` class. It might be useful, however, to move this particular `constexpr` global into `Operation` itself in another PR.